kratos/testing/test_case.h

```diff
@@ -24,6 +24,7 @@
 
 // Project includes
 #include "testing/tester.h"
+#include "containers/model.h"
```

Comment:
Do we need to include model here!?! 

It is breaking the dependencies in layers...